test/modules/aws-vpc-integration.ts

```diff
@@ -62,70 +77,85 @@ describe('VPC Integration Testing', () => {
 
   it('applies the subnet change', apply());
 
-  it('adds a new elastic ip', query(`
-    INSERT INTO elastic_ip (tags)
-    VALUES ('{"name": "${eip}"}');
-  `));
-
-  it('check elastic ip count', query(`
-    SELECT * FROM elastic_ip WHERE tags ->> 'name' = '${eip}';
-  `, (res: any) => expect(res.length).toBe(1)));
-
-  it('applies the elastic ip change', apply());
-
-  it('check elastic ip count', query(`
-    SELECT * FROM elastic_ip WHERE tags ->> 'name' = '${eip}';
-  `, (res: any) => expect(res.length).toBe(1)));
-
-  it('adds a new vpc', query(`  
-    INSERT INTO vpc (cidr_block)
-    VALUES ('192.${randIPBlock}.0.0/16');
-  `));
-
-  it('applies the vpc change', apply());
-
-  it('adds a private nat gateway', query(`
-    INSERT INTO nat_gateway (connectivity_type, subnet_id, tags)
-    SELECT 'private', id, '{"Name":"${ng}"}'
-    FROM subnet
-    WHERE cidr_block = '192.${randIPBlock}.0.0/16';
-  `));
-
-  it('applies the private nat gateway change', apply());
-
-  it('checks private nat gateway count', query(`
-    SELECT * FROM nat_gateway WHERE tags ->> 'Name' = '${ng}';
-  `, (res: any) => expect(res.length).toBe(1)));
-
-  it('adds a public nat gateway with existing elastic ip', query(`
-    INSERT INTO nat_gateway (connectivity_type, subnet_id, tags, elastic_ip_id)
-    SELECT 'public', subnet.id, '{"Name":"${pubNg1}"}', elastic_ip.id
-    FROM subnet, elastic_ip
-    WHERE cidr_block = '192.${randIPBlock}.0.0/16' AND elastic_ip.tags ->> 'name' = '${eip}';
-  `));
-
-  it('applies the public nat gateway with existing elastic ip change', apply());
-
-  it('checks public nat gateway with existing elastic ip count', query(`
-    SELECT * FROM nat_gateway WHERE tags ->> 'Name' = '${pubNg1}';
-  `, (res: any) => expect(res.length).toBe(1)));
-
-  it('adds a public nat gateway with no existing elastic ip', query(`
-    INSERT INTO nat_gateway (connectivity_type, subnet_id, tags)
-    SELECT 'public', subnet.id, '{"Name":"${pubNg2}"}'
-    FROM subnet
-    WHERE cidr_block = '192.${randIPBlock}.0.0/16';
-  `));
-
-  it('applies the public nat gateway with no existing elastic ip change', apply());
-
-  it('checks public nat gateway with no existing elastic ip count', query(`
-    SELECT * FROM nat_gateway WHERE tags ->> 'Name' = '${pubNg2}';
-  `, (res: any) => expect(res.length).toBe(1)));
-
-  it('checks public nat gateway with no existing elastic ip count', query(`
-    SELECT * FROM elastic_ip WHERE tags ->> 'Name' = '${pubNg2}';
-  `, (res: any) => expect(res.length).toBe(1)));
+  describe('Elastic IP and nat gateway creation', () => {
```

Comment:
I've created these inner `describe`s to separate the logic in the tests. NATs adds too much time to test execution. Having it in separate `describe`s let us comment what we do not want to execute then and quickly just run the new ones and make sure it is working without waiting that long locally.